cpp/src/parquet/properties.h

```diff
@@ -34,10 +34,14 @@
 
 namespace parquet {
 
+/// Control for data types in parquet.
 struct ParquetVersion {
   enum type { PARQUET_1_0, PARQUET_2_0 };
 };
 
+/// Controls layout of data pages.
```

Comment:
Could you add more background on the differences and why you'd want that?

Reply:
parquet-format v2.0.0 introduced a data page metadata and serialized page structure (for example, encoded levels are no longer compressed)

Reply:
I'll clarify, I'm asking this to be put in a doc comment for whomever read this and doesn't know which one to use.

Reply:
Added Wes's comment.